src/eredis_sentinel.erl

```diff
@@ -181,8 +184,8 @@ query_master(#eredis_sentinel_state{conn_pid=undefined,
             ) ->
     case eredis:start_link([{host, H},
                             {port, P},
-                            {username, Username},
-                            {password, Password},
+                            {username, deobfuscate(Username)},
+                            {password, deobfuscate(Password)},
```

Comment:
If eredis accepts `fun(() -> string())` too then this is not necessary. 

As mentioned in the issue `start_link` seems to be one of the things that generates "PROGRESS REPORT" entries in erlang logs. So we don't want its arguments to be plain text ideally.